lang/cs/index.md

```diff
@@ -13,18 +13,18 @@ Shrnutí
 Číslo verzí zapisujeme ve formátu MAJOR.MINOR.PATCH
 Navyšování jednotlivých čísel verzí probíhá následovně:
 
-1. MAJOR - když nastala změna, která není zpětně kompatibilní s ostatními (API)
-2. MINOR - když se přidá funkcionalita se zachováním zpětné kompatibility
+1. MAJOR – když nastala změna, která není zpětně kompatibilní s ostatními (API)
+2. MINOR – když se přidá funkcionalita se zachováním zpětné kompatibility
 3. PATCH - když se opravila chyba a zůstala kompatibilita
 
 Pomocí předběžných verzí a přidáváním metadat je možné upřesnit informace. Např.: `1.0.0-alfa`, `1.0.1-beta+2`
 
 Úvod
 ----
 
-Ve světě softwaru, jeho vývoje a správy existuje děsivé místo, kterému anglicky říkáme “dependency hell” (v překladu "peklo závislostí"). Čím větší a komplexnější váš projekt je, tím je větší šance, že se na tomto místě jednou ocitnete.
+Ve světě softwaru, jeho vývoje a správy existuje děsivé místo, kterému anglicky říkáme „dependency hell“ (v překladu „peklo závislostí“). Čím větší a komplexnější váš projekt je, tím je větší šance, že se na tomto místě jednou ocitnete.
```

Comment:
```suggestion
Ve světě softwaru, jeho vývoje a správy existuje děsivé místo, kterému anglicky říkáme "dependency hell" (v překladu "peklo závislostí"). Čím větší a komplexnější váš projekt je, tím je větší šance, že se na tomto místě jednou ocitnete.
```

Reply:
Could you use default doublequote `"` everywhere, to make it consistent? Thanks

Reply:
Sure, but it's incorrect to use English quotation marks (") here. In Czech we are using „“ instead. That's why I changed it.

Does it mean that you can somehow automagically replace default English marks with correct quotation marks (depending on various languages)? I'm not sure if this is a good approach.

Reply:
Ohhh.. So it's locale preferences, I didn't know. Learned something new this evening 💪 
I'll investigate in it to make some opinion, and will back with feedback